emsdk.py

```diff
@@ -1885,6 +1885,14 @@ def install_tool(self):
       print("Skipped installing " + self.name + ", already installed.")
       return True
 
+    version_hash = self.name
```

Comment:
this isn't necessarily a hash - for e.g. node it will be something like `node-VERSION`. How about `version_id`?

Reply:
Done